src/Controller/Api/FormController.php

```diff
@@ -38,8 +38,8 @@ public function validate(Request $request, Serializer $serializer, string $formT
         return new Response($serializer->serialize(
             [
                 'children' => array_filter($form->all(), function (FormInterface $form) {
-                    return 0 < $form->getErrors(true)->count();
```

Comment:
do you test this? I tried this and it

Reply:
Ok I just add a check if the form is submitted ^^,